app/models/post.rb

```diff
@@ -1,3 +1,15 @@
 class Post < ApplicationRecord
   belongs_to :user
+  has_many :likes
+  has_many :comments
+
+  def most_recent_five_comments
+    comments.includes(:user).order(created_at: :desc).limit(5)
+  end
```

Comment:
- [ ] Please add the following validations to your `Post` model as per requirements 👇 
- Title must not be blank.
- Title must not exceed 250 characters.
- CommentsCounter must be an integer greater than or equal to zero.
- LikesCounter must be an integer greater than or equal to zero.